FFXIVClientStructs/FFXIV/Client/Game/MirageManager.cs

```diff
@@ -0,0 +1,29 @@
+namespace FFXIVClientStructs.FFXIV.Client.Game;
+
+// ctor "48 89 5C 24 ?? 57 48 83 EC 20 48 8B D9 C6 01 00 48 83 C1 04"
+// note: data is cleared when switching zones
+[StructLayout(LayoutKind.Explicit, Size = 0x1478)]
+public unsafe partial struct MirageManager
+{
+    [StaticAddress("48 8B 1D ?? ?? ?? ?? 48 85 DB 74 48", 3, isPointer: true)]
+    public static partial MirageManager* Instance();
+
+    [FieldOffset(0)] public bool IsApplyingGlamourPlate;
+
+    [FieldOffset(0x4)] public fixed uint PrismBoxItemIds[4 * 800];
```

Comment:
i think you meant `uint PrismBoxItemIds[800]` here. 4 * 800 uints is too big (thats 4 * 800 * 4 bytes)

Reply:
Right, I was thinking in bytes. Thanks & fixed!